### PR TITLE
:memo: Redo project deletion for new flow

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

It was seen as too easy to delete projects, so a new flow was introduced. See additional context.

## What's changed

Updated the page for the new flow and added codetabs with the way in the CLI.

### CODE REVIEW

**Review by trolologuy — APPROVED** [review]
- [x] Final built pages are working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is clear.
- [x] The changes follow the content and markup style guides.

File: docs/src/administration/web/delete.md [hunk 1]
```diff
@@ -1,21 +1,47 @@
 ---
-title: "Delete a project"
+title: Delete a project
+description: See how to delete projects you no longer need.
 weight: 5
-sidebarTitle: "Deleting a project"
-toc: false
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/administration/web/delete.md:5]: It's an empty table of contents, so it shouldn't be displayed anyway. That's a bug that I fixed in some other PR, so I didn't think it was right to keep this here when it shouldn't be here.

File: docs/src/administration/web/delete.md [hunk 2]
```diff
@@ -1,21 +1,47 @@
...
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. Run the following command:
+
+   ```bash
+   platform project:delete --project <PROJECT_ID>
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/administration/web/delete.md:38]: In this case, I created an empty project and then deleted it.

[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/administration/web/delete.md:38]: Other way around -- changes in the console are upcoming

**Comment by CollierCZ** [comment]
Handled in #2237

**Pull request closed without merging**